### ISSUE TITLE
ArgumentError: unknown keyword: :aliases

### ISSUE THREAD

**Issue by ChrisCPO** [pawurb/rails-pg-extras#33]
When running the rake task locally I get an error instead of output. 

able to reproduce issue on:
ruby 2.7.6p219
Rails 6.0.4.7
gem version: 4.7.0

```
% rake pg_extras:diagnose
rake aborted!
ArgumentError: unknown keyword: :aliases

Tasks: TOP => pg_extras:diagnose => pg_extras:establish_connection
(See full trace by running task with --trace)
```
Using `safe_load` over `load` seems to address this, but I think there is a compatibility issues then.

**Comment by ChrisCPO** [comment]
https://github.com/pawurb/rails-pg-extras/pull/34

**Comment by pawurb** [comment]
Fix released in 4.7.1
